CreeDictionary/tests/API_tests/model_test.py

```diff
@@ -129,3 +126,47 @@ def test_search_for_stored_non_lemma():
     assert not result.initial_change_tags
     assert len(result.definitions) >= 1
     assert all(len(dfn.source_ids) >= 1 for dfn in result.definitions)
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("term", ["acâhkos kâ-osôsit", "acâhkosa kâ-otakohpit"])
+def test_search_space_characters_in_matched_term(term):
+    """
+    The search should find results with spaces in them.
+    See: https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/147
+    """
+
+    word = Wordform.objects.get(text=term)
+    assert word is not None
```

Comment:
⚠️ Please merge https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/tree/test-space-string -- there is a missing assertion that is a part of this test case.